perf/benchmarkapps/QpsWorker/worker-start-client.ps1

```diff
@@ -0,0 +1,56 @@
+# 1. Invoke the bidirection streaming method WorkerService/RunClient using grpcurl.
+# 2. Send a setup message to processes standard input.
+# 3. Leave standard input so the stream and client stay open.
+
+# Setup payload.
```

Comment:
add a note that the benchmark configuration is just for demonstration purposes, we can link to https://github.com/grpc/grpc/tree/master/tools/run_tests/performance#approach-1-use-grpc-oss-benchmarks-framework-recommended for how to obtain the specs for the "official" scenarios used for benchmarking gRPC.